extension/manifest.json

```diff
@@ -7,13 +7,13 @@
   "applications": {
     "gecko": {
       "id": "firefox-translations-addon@mozilla.org",
-      "strict_min_version": "79.0"
+      "strict_min_version": "90.0"
```

Comment:
If you're bumping the min supported version to 90, then this snippet for Firefox 87- can be removed: https://github.com/mozilla/firefox-translations/blob/61295c474e0e68d653b7aa561bececb78ab48c9a/extension/controller/experiments/TranslationBar/api.js#L10-L15